src/main/resources/META-INF/rewrite/add-common-annotations-dependencies.yml

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2024 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.javax.AddCommonAnnotationsDependencies
+displayName: Add explicit Common Annotations dependencies
+description: >
+  This recipe will add the necessary `annotation-api` dependency from Jakarta EE 8 to maintain compatibility with Java
+  version 11 or greater.
+tags:
+  - javax
+  - java11
+  - jsr250
+  - jakarta
+recipeList:
+  # Add or update the jakarta.annotation-api to a maven project. This artifact still uses the javax.annotation name space.
+  - org.openrewrite.java.dependencies.ChangeDependency:
+      oldGroupId: javax.annotation
+      oldArtifactId: javax.annotation-api
+      newGroupId: jakarta.annotation
+      newArtifactId: jakarta.annotation-api
+      newVersion: 1.3.x
+  - org.openrewrite.maven.ChangeManagedDependencyGroupIdAndArtifactId:
+      oldGroupId: javax.annotation
+      oldArtifactId: javax.annotation-api
+      newGroupId: jakarta.annotation
+      newArtifactId: jakarta.annotation-api
+      newVersion: 1.3.x
```

Comment:
```suggestion
```
`org.openrewrite.java.dependencies.ChangeDependency` delegates to `org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId`
https://github.com/openrewrite/rewrite-java-dependencies/blob/e15d0f1f5371e2bda660a7da56534c67412efd20/src/main/java/org/openrewrite/java/dependencies/ChangeDependency.java#L118-L120

And `ChangeDependencyGroupIdAndArtifactId` delegates to `ChangeManagedDependencyGroupIdAndArtifactId`
https://github.com/openrewrite/rewrite/blob/0324a3c60ac969bc00bce8b637021cb32248e82f/rewrite-maven/src/main/java/org/openrewrite/maven/ChangeDependencyGroupIdAndArtifactId.java#L159-L166

That means there's likely no need for this explicit recipe here.

Reply:
You might have seen this pattern elsewhere; we're in the middle of clearing those out in
- https://github.com/openrewrite/rewrite-migrate-java/pull/444/

Reply:
Indeed I copy pasted it from `org.openrewrite.java.migrate.javax.AddJaxbDependencies` here : https://github.com/openrewrite/rewrite-migrate-java/blob/v2.11.0/src/main/resources/META-INF/rewrite/java-version-11.yml#L113

I wasn't aware of that change. 
Thanks.